backend/open_webui/retrieval/utils.py

```
@@ -49,20 +60,63 @@
+    docs = loader.load()
+    content = " ".join([doc.page_content for doc in docs])
+    return content, docs
+
+
 class VectorSearchRetriever(BaseRetriever):
     collection_name: Any
     embedding_function: Any
     top_k: int
 
     def _get_relevant_documents(
+        self, query: str, *, run_manager: CallbackManagerForRetrieverRun
```

Ruff (0.15.0)

[warning] Unused method argument: `query`

(ARG002)